### DISCUSSION TITLE
carbonapi not getting the Graphite metrics stored in Victoria Metrics.

### DISCUSSION THREAD

**Discussion by ravivish83** [bookingcom/carbonapi#411] [General]
Hi,

I have requirement that I need to get the graphite data source metrics on to Grafana from Victoria Metrics(Graphite Datasource Metrics are stored in Victoria Metrics and pulling the metrics through graphite endpoint)  
I have found the one config in go-graphite  https://github.com/go-graphite/carbonapi/blob/main/cmd/carbonapi/carbonapi.example.victoriametrics.yaml
and which is working but it doesn't support the metrics with | (pipe character) to push to Grafana

I have recently found that https://github.com/bookingcom/carbonapi support the  | (pipe character) but I don't find the configuration support for Victoria Metrics .https://github.com/bookingcom/carbonapi/blob/master/config/carbonapi.yaml
 
Can you please kindly suggest me that the https://github.com/bookingcom/carbonapi fork support the Victoria Metrics (Graphite DataSource) ? if yes, please suggest the config settings for the same.? If not , any other workaround to make it available  ?

Thanks,

**Comment by deniszh** [comment]
Hello,
Unfortunately, booking.com fork of carbonapi do not support getting data from Victoria Metrics.
Not sure about pipe character, but you probably should open issue in go-graphite carbonapi for pipe support then.

**Reply by ravivish83** [reply]
Hi,

Thank you for the update .

Thanks
